stl/inc/xhash

```diff
@@ -1087,6 +1089,14 @@ public:
     size_type erase(const key_type& _Keyval) noexcept(
         _Nothrow_hash<_Traits, key_type> //
         && (!_Multi || _Nothrow_compare<_Traits, key_type, key_type>) ) /* strengthened */ {
+        return _Erase(_Keyval);
+    }
+
+private:
+    template <class _Keytype>
+    size_type _Erase(const _Keytype& _Keyval) noexcept(
+        _Nothrow_hash<_Traits, _Keytype> //
+        && (!_Multi || _Nothrow_compare<_Traits, key_type, _Keytype>) ) /* strengthened */ {

```

Comment:
Should we also check `_Nothrow_compare<_Traits, _Keytype, key_type>`, because in pathological cases the `noexcept`ness may depend on the parameter order? Occurs below.